app/controllers/notes/chapters/chapter_notes_controller.rb

```diff
@@ -25,9 +25,7 @@ def update
         @chapter_note = chapter.chapter_note.reload
         @chapter_note.assign_attributes(chapter_note_update_params.to_h)
 
-        if @chapter_note.valid?
-          @chapter_note.save
-
+        if @chapter_note.valid? && @chapter_note.save
```

Comment:
same here

Reply:
Yes, save should be enough, it performs the validation too: https://apidock.com/rails/ActiveRecord/Base/save
(there are other occurrences like this one)

Reply:
This isn't ActiveRecord? Its an API client using the `Her` gem?